src/pages/ServiceDetails/ServiceDetailsPage.tsx

```diff
@@ -321,12 +322,16 @@ class ServiceDetails extends React.Component<ServiceDetailsProps, ServiceDetails
           }
         });
     } else {
-      fetchTraces(this.props.match.params.namespace, this.props.match.params.service, getQueryJaeger())
+      const params = getQueryJaeger();
+      fetchTraces(this.props.match.params.namespace, this.props.match.params.service, params)
         .then(traces => {
+          const appendTraces = this.lastFetchTraceMicros !== undefined;
```

Comment:
I don't see how it would work, as `lastFetchTraceMicros` is never set.. am I missing something?

Reply:
not you are not missing nothing ...I forgot updated that value let me open another PR :( I checked that there were new points but i didn't see that the news are the same

Reply:
@aljesusg I see perhaps another issue: the traces fetched are never reset, which I think is a problem when query options change

Reply:
keeping old data was the point of this task

Reply:
Yes...I only tested refresh, because I calculate the from time that was the point...I'll reopen the issue